scripts/trento-support.sh

```diff
@@ -81,27 +81,26 @@ collect_kubernetes_state() {
 } &> "$OUTPUT"
 
 collect_all() {
-    collect_trento_configuration
-    collect_base_system
-    collect_kubernetes_state
+    collect_trento_configuration || echo "Error $? during 'configuration'"
```

Comment:
Maybe add a "collection" in the end of the sentences?
e.g.: "Error during configuration collection"